src/ui/components/ArchivedCredentials/ArchivedCredentials.test.tsx

```diff
@@ -80,7 +80,11 @@ describe("Archived and revoked credentials", () => {
         <ArchivedCredentialsContainer
           revokedCreds={[]}
           archivedCredentialsIsOpen={true}
-          archivedCreds={credsFixAcdc}
+          archivedCreds={credsFixAcdc.map((cred) => ({
+            ...cred,
+            credentialType: cred.s.title,
+            issuanceDate: cred.a.dt,
```

Comment:
None of the UI should be doing this. This PR has gone the wrong way :P

Remove the duplicated entries in `ACDCDetails` type, and stop the UI using them

Reply:
Sorry, but I see archivedCreds and revokedCreds are using the CredentialShortDetails type. That type has `credentialType` and `issuanceDate` as required fields, so what should we do?

Reply:
I understand that we omit credentialType and issuanceDate in ACDCDetails type because it duplicates other fields and is not returned to UI anymore

Reply:
The `archivedCreds` prop of `ArchivedCredentialsContainer` is `CredentialShortDetails[]` not `ACDCDetails[]`. This is fetched with `getCredentials` in `Credentials.tsx` and put into Redux.

I'm asking to remove these fields from `ACDCDetails` - why does that affect `CredentialShortDetails`?

Reply:
we remove the field of ACDCDetails in the interface, right? like 
`interface ACDCDetails extends Omit<CredentialShortDetails, "credentialType" | "issuanceDate">`

Reply:
yes this is what I was thinking